cdk/src/open-data-platform/network/network-stack.ts

```diff
@@ -15,44 +15,22 @@ export class NetworkStack extends Stack {
   constructor(scope: Construct, id: string, props: CommonProps) {
     super(scope, id, props);
 
-    const { envType } = props;
-
-    // This is currently set to always use a new VPC, since we don't yet have a good way to
-    // determine whether to use a new or existing one (and can't dynamically figure that out [1]).
-    // [1] https://github.com/aws/aws-cdk/issues/5305#issuecomment-565394725
-    this.vpc = [EnvType.Development, EnvType.Sandbox].includes(envType)
-      ? // Use an existing VPC.
-        // By default, an AWS account can only have 5 Elastic IP addresses. Reusing existing VPCs keeps
-        // that number down.
-        ec2.Vpc.fromLookup(this, id, {
-          // tags: {
-          //   Project: projectName,
-          // },
-
-          // Temporarily hardcode the ID since the tag lookup was not finding the right VPC.
-          // TODO: replace with search by tag.
-          vpcId: 'vpc-03af3621549e79f22',
-          region: 'us-east-2',
-        })
-      : // Else, create a new VPC.
-        // Note that changes here will not be reflected on deployment if a VPC already exists.
-        // TODO: document how to make a change to an existing VPC.
-        new ec2.Vpc(this, 'VPC', {
-          cidr: '10.0.0.0/16',
-          maxAzs: 2,
-          subnetConfiguration: [
-            {
-              cidrMask: 24,
-              name: 'public',
-              subnetType: ec2.SubnetType.PUBLIC,
-            },
-            {
-              cidrMask: 20,
-              name: 'private',
-              subnetType: ec2.SubnetType.PRIVATE_WITH_NAT,
-            },
-          ],
-        });
+    this.vpc = new ec2.Vpc(this, 'VPC', {
+      cidr: '10.0.0.0/16',
+      maxAzs: 2,
```

Comment:
Have maxAzs = 1 when not in Production. We only need the availability achieved by using mulitiple AZs for the env that users interact with. It's ok if Dev breaks because of an AWS AZ outage

Reply:
Set nat gateways to `1` in non-prod envs 👍🏾 nice $catch ty